package-lock.json

```diff
@@ -1,9928 +1,7 @@
 {
```

Comment:
This is an odd. Another PR recently also wiped out the `package.lock` file. I wonder why this happened in both cases... Please restore the lock file.